kratos/tests/test_model_part_io.py

```diff
@@ -237,7 +240,7 @@ def test_model_part_io_read_model_part_mesh_only(self):
         self.assertEqual(properties_1[KratosMultiphysics.LOCAL_INERTIA_TENSOR][2,2], 0)
 
     @KratosUnittest.skipUnless(structural_mechanics_is_available,"StructuralMechanicsApplication is not available")

```

Comment:
We could remove this line right?

Reply:
Nope, this prevents to fail in case StructuralMechanicsApplication is not compiled

Reply:
My point is that in these tests the StructuralApp is not used, so it shouldn't be needed to have the StructuralApp compiled right?

https://github.com/KratosMultiphysics/Kratos/blob/2582922cdab863aad837036d80c087a0d8f68cb4/kratos/tests/test_model_part_io.py#L242-L270

Reply:
There is CL from the StructuralMechanicsApplciation, it is used indirectly